### ISSUE TITLE
Angular Template can't be run

### ISSUE THREAD

**Issue by NorDroN** [aspnet/JavaScriptServices#1621]
Hi! I've created new angular app and when I run my app I always see:
```
TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 50 seconds. Check the log output for error information.
```
I've tried to set Timeout more 5 minutes, but it doesn't help.

**Comment by NorDroN** [comment]
When I run my app I see
```
...
Now listening on: http://127.0.0.1:28660
Application started. Press Ctrl+C to shut down.
...
Request starting HTTP/1.1 GET http://localhost:44365/
...
** NG Live Development Server is listening on localhost:35285, open your browser on http://localhost:35285/ **
...
webpack: Compiled successfully.
...
```
and browser open page https://localhost:44365/

Maybe problem with https?

PS: If I open http://localhost:35285/ than I can see client part of my app

**Comment by martinstenhoff** [comment]
Perhaps you are hitting #1611.
I saw the same thing whenever I launched using F5 (debugger attached). If I launched without debugger the app would start and I could then attach the debugger afterwards.

**Comment by SteveSandersonMS** [comment]
The issue with running under the debugger was resolved in the 2.1.0 RC release. Hopefully your problem is now resolved, but if it's still occurring (after updating to 2.1.0 final), please let us know.
